src/box/lua/connpool.lua

```diff
@@ -232,12 +237,21 @@ local function filter(opts)
         roles = '?table',
         sharding_roles = '?table',
         mode = '?string',
+        skip_connection_check = '?boolean'
     })
     opts = opts or {}
+
     if opts.mode ~= nil and opts.mode ~= 'ro' and opts.mode ~= 'rw' then
         local msg = 'Expected nil, "ro" or "rw", got "%s"'
         error(msg:format(opts.mode), 0)
     end
+
+    if opts.skip_connection_check and opts.mode ~= nil then
+        local msg = 'Filtering by mode "%s" requires the connection ' ..
+                    'check disabled by the "skip_connection_check" option'
+        error(msg:format(opts.mode), 0)
+    end
```

Comment:
I guess you meant the opposite: the connection checking functonality is required to perform filtering by the mode, so it can't be disabled in this case.

Reply:
Fixed. Thank you. I've meant something like `Filtering by mode "%s" requires the connection check [that has been] disabled by the "skip...`. I've added the lacking words from the square brackets.